src/platforms/common/performance/index.mdx

```diff
@@ -134,6 +135,23 @@ If you are using `react-router`, we provide <PlatformLink to="/configuration/int
 
 </PlatformSection>
 
+<PlatformSection supported={["php"]}>
+
+## Improve Response Time
+
+Using the performance capabilities of PHP has some impact on your response time (depending on the `traces_sample_rate` you configured).
+Because of the nature of PHP, requests in most cases are sent in the same process as you serve your users' response.
+In sum, this process affects the time it takes to send a request to Sentry, as it is added on top of your servers' response time. To mitigate this and reduce the addition close to zero, you can use a Relay running locally on the same machine or local network that acts as a proxy/agent. As a result, instead of your PHP process sending things to Sentry, the PHP process sends it to your local Relay, which then forwards it to Sentry.
```

Comment:
```suggestion
In short, this process affects the time it takes to send a request to Sentry, because it is added to your servers' response time. 

To account for this and reduce this addition to close to zero, you can use [Relay](/product/relay/getting-started/) running locally on the same machine or local network that acts as a proxy/agent. As a result, instead of your PHP process sending things to Sentry, the PHP process sends it to your local Relay, which then forwards it to Sentry.
```